spec/at-rules/import.md

```diff
@@ -6,6 +6,23 @@ still supported for backwards-compatibility.
 
 [`@use` rule]: use.md
 
+## Table of Contents
+
+* [Syntax](#syntax)
+* [Semantics](#semantics)
+
+## Syntax
+
+<x><pre>
+**ImportRule**                ::= '@import' ImportArgument (',' ImportArgument)*
+**ImportArgument**            ::= ImportUrl ImportSupportsDeclaration? [MediaQueryList][]?
```

Comment:
It's probably a good idea to update the semantics below to explicitly refer to `ImportSupportsDeclaration` and `MediaQueryList` rather than the looser "supports query" and "media query".